packages/grafana-test-utils/src/utils/features.ts

```diff
@@ -0,0 +1,49 @@
+import { FeatureToggles } from '@grafana/data';
+import { config } from '@grafana/runtime';
+
+/**
+ * Enables and disables feature toggles `beforeEach` test, and sets back to original settings `afterEach` test
+ */
+export const testWithFeatureToggles = ({
+  enable,
+  disable,
+}: {
+  enable?: Array<keyof FeatureToggles>;
+  disable?: Array<keyof FeatureToggles>;
+}) => {
+  const originalToggles = { ...config.featureToggles };
+
+  beforeEach(() => {
+    enable?.forEach((featureToggle) => {
+      config.featureToggles[featureToggle] = true;
+    });
+    disable?.forEach((featureToggle) => {
+      config.featureToggles[featureToggle] = false;
+    });
```

Comment:
can we use a `for-of` loop instead of `arr.forEach` that modifies an object outside of it's cb scope?